Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS builder
 WORKDIR /app
 
 # Copy the whole solution into the first stage
-COPY * ./server/
```

Comment:
@tonyo I think this flattens the structure when copying. I used this elsewhere and broke stuff.
So I guess I need to change to `./` or something although now it breaks this GCB :)

Reply:
GCB shouldn't do anything weird here, you can just test `docker build` locally.